### DISCUSSION TITLE
Welcome to aiohttp Discussions!

### DISCUSSION THREAD

**Discussion by webknjaz** [aio-libs/aiohttp#4961] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  P.S. Bugs should still be reported as issues.

**Comment by pranpraveen** [comment]
Hello Gents,

I am getting below error with aiohttp. Anyone else faced same error? Appreciate any help or advice.

 building 'aiohttp._websocket' extension
      creating build\temp.win-amd64-cpython-311
      creating build\temp.win-amd64-cpython-311\Release
      creating build\temp.win-amd64-cpython-311\Release\aiohttp
      "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\pranp\AppData\Local\Programs\Python\Python310\include -IC:\Users\AppData\Local\Programs\Python\Python310\Include "-IC:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\include" "-IC:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22000.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22000.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22000.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22000.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22000.0\\cppwinrt" /Tcaiohttp/_websocket.c /Fobuild\temp.win-amd64-cpython-311\Release\aiohttp/_websocket.obj
      _websocket.c
      aiohttp/_websocket.c(1716): warning C4047: 'function': 'PyObject *' differs in levels of indirection from 'int'
      aiohttp/_websocket.c(1716): warning C4024: 'PyCode_New': different types for formal and actual parameter 14
      aiohttp/_websocket.c(1716): warning C4047: 'function': 'int' differs in levels of indirection from 'PyObject *'
      aiohttp/_websocket.c(1716): warning C4024: 'PyCode_New': different types for formal and actual parameter 15
      aiohttp/_websocket.c(1716): error C2198: 'PyCode_New': too few arguments for call
      aiohttp/_websocket.c(1944): error C2027: use of undefined type '_frame'
      C:\Users\pranp\AppData\Local\Programs\Python\Python310\include\pytypedefs.h(22): note: see declaration of '_frame'
      aiohttp/_websocket.c(1944): error C2037: left of 'f_localsplus' specifies undefined struct/union '_frame'
      aiohttp/_websocket.c(1944): error C2056: illegal expression
      aiohttp/_websocket.c(1944): error C2466: cannot allocate an array of constant size 0
      aiohttp/_websocket.c(1944): warning C4034: sizeof returns 0
      aiohttp/_websocket.c(2306): error C2039: 'recursion_depth': is not a member of '_ts'
      C:\Users\pranp\AppData\Local\Programs\Python\Python310\include\cpython/pystate.h(82): note: see declaration of '_ts'
      aiohttp/_websocket.c(2308): error C2039: 'recursion_depth': is not a member of '_ts'
      C:\Users\pranp\AppData\Local\Programs\Python\Python310\include\cpython/pystate.h(82): note: see declaration of '_ts'
      aiohttp/_websocket.c(2335): error C2065: 'CO_NOFREE': undeclared identifier
      aiohttp/_websocket.c(2760): warning C4047: 'function': 'PyObject *' differs in levels of indirection from 'int'
      aiohttp/_websocket.c(2744): warning C4024: 'PyCode_New': different types for formal and actual parameter 14
      aiohttp/_websocket.c(2760): warning C4047: 'function': 'int' differs in levels of indirection from 'PyObject *'
      aiohttp/_websocket.c(2744): warning C4024: 'PyCode_New': different types for formal and actual parameter 15
      aiohttp/_websocket.c(2744): error C2198: 'PyCode_New': too few arguments for call
      aiohttp/_websocket.c(2791): error C2037: left of 'f_lineno' specifies undefined struct/union '_frame'
      error: command 'C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Tools\\MSVC\\14.35.32215\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for aiohttp
Failed to build aiohttp
ERROR: Could not build wheels for aiohttp, which is required to install pyproject.toml-based projects
